src/components/Login/Login.js

```diff
@@ -59,6 +60,18 @@ class Login extends Component {
     this.props.login(this.state.identifier, this.state.username);
   };
 
+  fetchData = () => this.props.fetchData();
+
+  getUsers() {
+    var usersData = [];
+    usersData.push(<Picker.Item key={999} label={strings.user} value={null} />);
+    this.props.dataFetch.map((user, identifier) => {
```

Comment:
Remember that `map` returns a new array that each elements is the result of applying the function. I believe that what you actually want to use is a `forEach` here